include/benchmark/benchmark_api.h

```diff
@@ -445,7 +445,7 @@ class Benchmark {
 // BENCHMARK_TEMPLATE(BM_Foo, 1);
 //
 // will register BM_Foo<1> as a benchmark.
-#define BENCHMARK_TEMPLATE(n, a)                             \
+#define BENCHMARK_TEMPLATE1(n, a)                            \
```

Comment:
They don't need to be but I see value in keeping them defined. If I have a `BM_foo<T, U>` in `foo.cpp` and I don't know if `foo.cpp` will be compiled in C++11 then I would want to use `BENCHMARK_TEMPLATE2` so that it always works.